src/import_edf.jl

```diff
@@ -208,81 +267,262 @@ function groupby(f, list)
     return d
 end
 
+# unpack a single channel spec from labels:
+# "channel"
+canonical_channel_name(channel_name) = channel_name
+# "channel" => ["alt1", "alt2", ...]
+canonical_channel_name(channel_alternates::Pair) = first(channel_alternates)
```

Comment:
I think this is mostly clear from the example above, but it would be good to have in the docs/docstrings somewhere that the specification is `canonical_name => [alt1, alt2]`. In other words, just documenting this behavior in a way that's discoverable without looking at the source.

Reply:
got a bit further and saw that this is documented